### DISCUSSION TITLE
Gnome Wayland question

### DISCUSSION THREAD

**Discussion by Larson22-edu** [BenWestgate/bitcoin-core-on-tails#4] [Q&A] [answered]
Hello, I'm very excited about this project, but I do not know how to communicate on GitHub so forgive me if I am asking for help in the wrong place.

I am downloading as instructed, but have been stopped by the following error in the terminal:

**amnesia@amnesia:~$ Warning: Ignoring XDG_SESSION_TYPE=wayland on Gnome. Use QT_QPA_PLATFORM=wayland to run on Wayland anyway.
qt.qpa.plugin: Could not find the Qt platform plugin "wayland" in ""
This application failed to start because no Qt platform plugin could be initialized. Reinstalling the application may fix this problem.

Available platform plugins are: xcb.**

What should I do in this scenario? Again, thank you for this very exciting project, and let me know how I can better communicate, and how I can help the project in any way.

LL

**Comment by BenWestgate** [comment, marked as answer]
Are you using Tails 5.8?

It switched the window manager from 5.7 and it may be related to this error.

I haven't tested Tails 5.8 yet with this script. I've been working on an all graphical one that doesn't need admin thanks to the new persistent storage design in Tails 5.8

**Reply by Larson22-edu** [reply]
Yes, I am using Tails 5.8

That sounds great, I'll just keep an eye on this in the meantime. If I was to tell others about this project, where would you like me to direct them?

**Reply by BenWestgate** [reply]
https://github.com/BenWestgate/bitcoin-core-on-tails

will remain an alpha forever since Tails 5.8 made persistence simpler to setup without a restart.

I was able to remove 50% of the code in the new edition that'll I'll call "Bails"

That will be a beta as soon as it's done because I made a way to easily make backup Tails USBs making it safe to actually use and not missing features.

And after independent review by a couple experts, assuming no problems, release it as v1.0.0.


I made a merge request for my code that creates new backup Tails USBs without restarting or admin:
https://gitlab.tails.boum.org/tails/tails/-/merge_requests/993

They've started to look at it, although whether it gets into Tails or Not, "Bails" will have the feature because the code works

**Reply by BenWestgate** [reply]
Direct them to my Twitter dms, or if they're developers to this repository to check out this outdated alpha version.

I should have a Beta of its successor 'Bails' out in a week or less.

While the respin is almost done I haven't tried launching Bitcoin Core in Tails 5.8 yet to see how to avoid the error you've reported.

**Reply by BenWestgate** [reply]
https://tails.boum.org/news/version_5.8/#index2h3

It's a Known issue with Tails 5.8 and Qt applications, they have published a workaround on their website.

**Comment by BenWestgate** [comment]
If you could launch Tails with an admin password and persistence enabled, then

https://gitlab.tails.boum.org/tails/tails/-/merge_requests/993

Download the 3 files of this merge request, replace the 2 system file with the downloaded versions and add the third and then run: "Backup Tails" while screen recording and upload the demo videos that would help my code to directly make backup Tails USBs get into Tails and help thousands of people.

**Reply by BenWestgate** [reply]
Scratch this. I did it just now. Will upload the video tonight so Tails team can review the UX.
